tools/vms-generator/utils/utils.go

```diff
@@ -456,9 +465,11 @@ func GetVMIMasquerade() *v1.VirtualMachineInstance {
 	vm.Spec.Domain.Resources.Requests[k8sv1.ResourceMemory] = resource.MustParse("1024M")
 	vm.Spec.Networks = []v1.Network{{Name: "testmasquerade", NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}}}}
 	initFedora(&vm.Spec)
-	userData := "#!/bin/bash\necho \"fedora\" |passwd fedora --stdin\nyum install -y nginx\nsystemctl enable --now nginx"
 	networkData := "version: 2\nethernets:\n  eth0:\n    addresses: [ fd10:0:2::2/120 ]\n    dhcp4: true\n    gateway6: fd10:0:2::1\n"
```

Comment:
Would be nice to have generate string func for network data as well, it would make it more consistent.

Reply:
I'm torn; the goal of the PR is to "replace bash scripts with cloud-config on cloud-init user data" . 

You want to increase the scope, but the only benefit is code consistency (on user data, we have more portable examples, which is functionally preferable).

I think this belongs on a follow up PR.